### DISCUSSION TITLE
Select colors from .gpl palette

### DISCUSSION THREAD

**Discussion by ghost** [akiraux/Akira#596] [Ideas]
`.gpl` files are text files which does not require hard code to read as reading a binary file
its syntax is so simple
```
GIMP Palette
Name: Color Plate
Columns: 22
#
  0   0   0  BLACK
 26  26  26  90% GRAY
 51  51  51  80% GRAY
 77  77  77  70% GRAY
102 102 102  60% GRAY
128 128 128  50% GRAY
153 153 153  40% GRAY
179 179 179  30% GRAY
204 204 204  20% GRAY
230 230 230  10% GRAY
236 236 236  7.5% GRAY
242 242 242  5% GRAY
249 249 249  2.5% GRAY
255 255 255  WHITE
000 000 000 DIVIDER
244 067 054 #F44336
255 235 238 #FFEBEE
255 205 210 #FFCDD2
239 154 154 #EF9A9A
229 115 115 #E57373
239 083 080 #EF5350
```
the real problem is not reading the file but how will the user import them and after getting imported where should they appear

**Comment by ghost** [comment]
@Alecaddd, @LucileDT : Any ideas on where the imported colors should appear

**Reply by LucileDT** [reply]
I think it could be nice to do something similar to Krita with its Palette management system: https://docs.krita.org/en/reference_manual/dockers/palette_docker.html
